### PR TITLE
Leaderboard: hit the API

### PR DESCRIPTION
**Pull request by benjp009**
# 👋 Micronaut Reviewer 

[Leaderboard: hit the API](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/m2_send_receive_data.md)

### Project requirements

- [x] Read the Leaderboard API documentation to learn how to use this API.
- [x] Create a new game with the name of your choice by using the API.
- [x] Implement the "Refresh" button (receiving data from the API and parsing the JSON).
- [x] Implement the form "Submit" button (sending data to the API).
- [x] Use async and await JavaScript features to consume the API.
- [x] No styling is required.

### Pair Programming 🎳

For this exercise, PP was not equired.

Happy review :)

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1 +1,42 @@
 import './style.css';
+
+async function sendData(url = '', data = {}) {
```
Comment (bimbolabuari) [inline src/index.js]: Let's use the ES6 arrow function here instead of the `function` keyword ✊

**Review by bimbolabuari — CHANGES_REQUESTED** [review]
Hi @benjp009 , 

You have done a good job so far 🚀  

## Highlights 🎉 

- Implement the "Refresh" button
- Implement the form "Submit" button
- Use async and await JavaScript features to consume the API.

There are some issues that you still need to work on to go to the next project but you are almost there! 🎉 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

--------


_Reviewed by  @bimbolabuari_

File: src/index.js (hunk 1 above)
Comment (benjp009) [reply src/index.js:3]: @bimbolabuari please explain what do you mean since the code is the same as the exercise offered by Microverse on mozilla.org

Comment (benjp009) [reply src/index.js]: Changes have been done - thanks

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @benjp009,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by benjp009** [comment]
@TedLivist Thanks :)

**Pull request merged**